index.html

```diff
@@ -1,155 +1,206 @@
-            </div>
-            <div class="rightSection">
-                <img src="/images/Api Programming logo.png" alt="Open Code Era logo">
-            </div>
-        </section>
-        <hr>
-        <section class="secondSection" id="about">
-
-            <div class="second-section-container">
-                  <div class="leftSection">
+                </div>
+                <div class="rightSection">
+                <img
+                    src="/images/Api Programming logo.png"
+                    alt="Open Code Era logo"
+                />
+                </div>
+            </section>
+            <hr />
+            <section class="secondSection" id="about">
+                <div class="second-section-container">
+                <div class="leftSection">
                     <!-- image from template.net -->
-                    <img src="images/illustrations/goals.png" alt="goals">
-            </div>
+                    <img src="images/illustrations/goals.png" alt="goals" />
+                </div>
 
-                <div class="rightSection"> 
+                <div class="rightSection">
                     <h2>Our Mission</h2>
                     <span class="text-gradient-blue-white">
-                        <p>Open Code Era fosters open source culture, encouraging collaboration in software development within the developer community.</p>
+                    <p>
+                        Open Code Era fosters open source culture, encouraging
+                        collaboration in software development within the developer
+                        community.
+                    </p>
                     </span>
-                </div> 
-            </div>
+                </div>
+                </div>
+            </section>
 
-        </section>
-        <hr>
-        <section class="contactSection" id="contact">
-            <div class="leftContact">
+            <hr />
+
+            <section class="contactSection" id="contact">
+                <div class="leftContact">
                 <h2>Get in Touch!</h2>
-                <p>If you have any questions or inquiries, please feel free to get in touch with us.</p>
+                <p>
+                    If you have any questions or inquiries, please feel free to get in
+                    touch with us.
+                </p>
                 <!-- image from freepik -->
-                <img src="./images/illustrations/Get in touch.png" alt="Get In Touch">
-            </div>
-            <div class="rightContact">
-                <form id="messageForm">
-                    <input type="text" id="name" name="name" placeholder="Name" >
-    
-                    <input type="text" id="email" name="email" placeholder="Email" >
-    
-                    <textarea id="message" name="message" rows="8" placeholder="Message"></textarea>
-    
-                    <button id="submit" type="submit" onclick="toggle('body')">Submit</button>
+                <img
```

Comment:
Avoid formatting the part of code that you are not working on, as this leads to merge conflicts on other contributor's PR.

You can open another PR to properly format the code.

Reply:
Okay, I will keep in mind for future contributions.